shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -229,7 +229,7 @@ void PlatformViewChannel::OnResize(
   }
   view->Resize(*width, *height);
 
-  result->Success();
+  result->Success(*arguments);
```

Comment:
Don't you need some logic as below?

```java
            final PlatformViewBufferSize sz = handler.resize(resizeRequest);
            if (sz == null) {
              result.error("error", "Failed to resize the platform view", null);
            } else {
              final Map<String, Object> response = new HashMap<>();
              response.put("width", (double) sz.width);
              response.put("height", (double) sz.height);
              result.success(response);
            }
```

Reply:
You are right, but in the current state, I guess we don't need it. 
It is true that readability is poor.